src/lib/components/specialized/service-search.svelte

```diff
@@ -105,7 +105,10 @@
       cityCode = props.citycode;
       cityLabel = getCityLabel(address);
       label = getAddressLabel(address);
-      [lon, lat] = address.geometry.coordinates;
+      const entireCity = address.properties.label === address.properties.city;
```

Comment:
ça m’a l’air un peu fragile, est-ce qu'on peut pas plutot faire 
```suggestion
      const entireCity = address.properties.type === "municipality";
``` 
?
(voir https://adresse.data.gouv.fr/api-doc/adresse)

Reply:
Bien vu, j'avais loupé cet attribut ! 57b8b0239500ae31604aa396a3fe59e1e551a398